lib/mp4/transmuxer.js

```diff
@@ -112,11 +130,9 @@ AudioSegmentStream = function(track) {
     collectDtsInfo(track, data);
 
     if (track) {
-      track.audioobjecttype = data.audioobjecttype;
-      track.channelcount = data.channelcount;
-      track.samplerate = data.samplerate;
-      track.samplingfrequencyindex = data.samplingfrequencyindex;
-      track.samplesize = data.samplesize;
+      audioProperties.forEach(function(prop) {
+        track[prop] = data[prop] || '';
```

Comment:
previously we did not do anything with this information, and from what I have seen it is always defined. If it was not defined the muxer probably would not even be able to get this far. So we could probably forgo the default empty string if we wanted to.